Makefile

```diff
@@ -251,7 +251,7 @@ build-tf2-cpu: build-cpu-py-38-base
 	    --platform "$(PLATFORMS)" \
 		--build-arg BASE_IMAGE="$(DOCKERHUB_REGISTRY)/$(CPU_PY_38_BASE_NAME)-$(SHORT_GIT_HASH)" \
 		--build-arg TENSORFLOW_PIP="tensorflow-cpu==2.8.3" \
-		--build-arg TORCH_PIP=$(TORCH_PIP_VERSION_TF2) \
+		--build-arg TORCH_PIP="$(TORCH_PIP_TF2_CPU)" \
```

Comment:
This is where you should remove the double quotes. (I initially thought it would be useful to remove them in the definition of TORCH_PIP_TF2_xxx)